ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -317,6 +321,8 @@ - (NSURL *)composeBrokerRequest:(ADAuthenticationError* __autoreleasing *)error
       BROKER_MAX_PROTOCOL_VERSION : @"2",
       @"extra_qp"       : _queryParams ? _queryParams : @"",
       @"claims"         : _claims ? _claims : @"",
+      @"enrollment_ids" : enrollmentIds ? enrollmentIds : @"",
```

Comment:
nit: @jasoncoolmax can we call "enrollment_ids" "intune_enrollment_ids" or "ca_enrollment_ids" here? If we were to publish the broker protocol, what would be easier to understand?

Reply:
I like "intune_enrollment_ids" better. 
But I think @midella may know better about which name might be more precise :)

Reply:
intune_enrollment_ids is more precise